DistroLauncher/AppStrategyCommon.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
```

Comment:
I'd stay consistent with naming and call these `ApplicationStrategyCommon.(h|cpp)` (or change others to use `App` as well).